dizoo/mujoco/config/ant_ppo_config.py

```diff
@@ -19,7 +19,6 @@
         model=dict(
             obs_shape=111,
             action_shape=8,
-            continuous=True,
```

Comment:
why remove `continuous` here

Reply:
changed 'continuous=True' with 'action_space='continuous'' in all ppo/onppo/trex configs in mujoco